nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/test/java/org/nuxeo/elasticsearch/test/nxql/TestNxqlConversion.java

```diff
@@ -59,8 +59,9 @@
  */
 @RunWith(FeaturesRunner.class)
 @Features({RepositoryElasticSearchFeature.class})
+@Deploy("org.nuxeo.elasticsearch.core:elasticsearch-test-contrib.xml")
```

Comment:
Why changing it from `org.nuxeo.elasticsearch.core.test` to `org.nuxeo.elasticsearch.core`?